eureka/S5_lightcurve_fitting/lightcurve.py

```diff
@@ -198,11 +198,18 @@ def plot(self, meta, fits=True):
         # Draw the data
         ax = fig.gca()
         ax.errorbar(self.time, self.flux, self.unc, fmt='.', color='w', ecolor=self.color, mec=self.color, zorder=0)
+        
+        # Make a new color generator for the models
+        plot_COLORS = color_gen('Category10')
```

Comment:
The intention is to avoid different colored model lines on colored data points because each color is associated with a specific wavelength. I suggest using 'Greys9' instead for the models.